mondoo.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh -e
+printf "\033[0;31mMondoo command is deprecated. Use cnspec instead.\033[0m\n"
+cnspec "$*"
```

Comment:
Is there a particular reason you used $* over the more common $@?

Reply:
after reading about the difference between the two, it certainly appears that `$@` is more appropriate to use. changed.